src/software/simulation/simulator.cpp

```diff
@@ -126,6 +129,8 @@ void Simulator::stepSimulation(const Duration& time_step)
     }
 
     physics_world.stepSimulation(time_step);
+
+    frame_number++;
```

Comment:
Should we step on simulation, or on `getSsLWrapperPacket`?

Reply:
simulation. Otherwise multiple consumers of the same simulator would get different frame numbers, or a single consumer could get the same state with different frame numbers, which I'm assuming we don't want.